blocks/bricks/sequence_generators.py

```diff
@@ -439,7 +462,8 @@ class SoftmaxEmitter(AbstractEmitter, Initializable, Random):
     Interprets readout elements as energies corresponding to their indices.
 
     """
-    def _probs(self, readouts):
+    @application
+    def emit_probs(self, readouts):
```

Comment:
Can we call it just `probs`? I know I have not really documented or even explain terminology, but only outputs are emitted, because there stochasticity may be involved. All the rest is computed deterministically.